### ISSUE TITLE
consider using testnet4

### ISSUE THREAD

**Issue by melvincarvalho** [thunderbiscuit/padawan-wallet#376]
It's awesome!

https://testnet4.com/

**Comment by thunderbiscuit** [comment]
Yes I'm keeping an eye on that for sure. I just published a new version of the app that uses Signet instead of Testnet3, which makes it usable again. I also got a donation of Signet coins, so I'm able to send more (150,000 satoshis) on app startup.

At some point if Testnet4 becomes the default and more used network I'll consider switching (I'll need to see if it would be better than Signet for some reason).

Thanks for opening the issue! It's a good place to keep track of the conversation around it.

**Comment by melvincarvalho** [comment]
Great!  I think it may get merged soon, the BIP was worked on today.  Testnet simulates bitcoin quite well, because it has mining, variablility, difficulty and so on.  It's running smoothly so far.  How much of a donation do you need?  It might be possible to get some testnet4 coins to you, too.

**Comment by thunderbiscuit** [comment]
Totally! Once it's up and running for a while I'll reach out to folks who mine and should be able to get my hands on some (or I'll ping you maybe?).

**Comment by melvincarvalho** [comment]
Sounds good!  I made this too:

https://github.com/testnet4/awesome-testnet4

**Comment by melvincarvalho** [comment]
Good news!  testnet4 is now merged into bitcoin master.  Let me know how many coins you need to seed your wallets, and an address, and I will see if I can find some for you.

https://github.com/bitcoin/bitcoin/pull/29775

**Comment by garyray-k** [comment]
I also have a grip of coins available for testnet4.

**Comment by thunderbiscuit** [comment]
Thanks guys! 

I think eventually Testnet 4 might be where I end up with Padawan if the faucet situation doesn't improve for Signet. 

But just to give an update of where things stand at the moment, Testnet 4 is just too young, not enabled in most bitcoin libraries, and the upshot of migrating the whole app from Signet to Testnet 4 is not very strong. I'm parking this for now; will likely revisit in a year or so?

If you have live faucets for Testnet 4 please post them here so I can keep a list of them!

Cheers.
